microprofile/microprofile-fault-tolerance-2.0/src/main/java/org/wildfly/swarm/ts/microprofile/fault/tolerance/v20/priority/AfterFT.java

```diff
@@ -0,0 +1,33 @@
+package org.wildfly.swarm.ts.microprofile.fault.tolerance.v20.priority;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import javax.annotation.Priority;
+import javax.inject.Inject;
+import javax.interceptor.AroundInvoke;
+import javax.interceptor.Interceptor;
+import javax.interceptor.InterceptorBinding;
+import javax.interceptor.InvocationContext;
+
+@Interceptor
+@Priority(6000)
+@AfterFT.InterceptAfter
+public class AfterFT {
+    @Inject
+    private InterceptorsContext context;
+
+    @Retention(RetentionPolicy.RUNTIME)
+    @Target({ElementType.TYPE, ElementType.METHOD})
+    @InterceptorBinding
+    public @interface InterceptAfter {
```

Comment:
Move the annotations outside of the interceptor classes? Otherwise the words "after" and "before" are duplicated on use sites (`AfterFT.InterceptAfter`).

Reply:
OK, separated.